src/index.html

```diff
@@ -1,14 +1,229 @@
+      discrimination in violation of this Declaration and against any incitement to such
+      discrimination.
+    </p>
+    <h2 class = "articles">
+      Article 8
+    </h2>
+    <p>
+      Everyone has the right to an effective remedy by the competent national tribunals
+      for acts violating the fundamental rights granted him by the constitution or by law.
+    </p>
+    <h2 class = "articles">
+      Article 9
+    </h2>
+    <p>
+      No one shall be subjected to arbitrary arrest, detention or exile.
+    </p>
+    <h2 class = "articles">
+      Article 10
+    </h2>
+    <p>
+      Everyone is entitled in full equality to a fair and public hearing by an independent
+      and impartial tribunal, in the determination of his rights and obligations and of any
+      criminal charge against him.
+    </p>
+    <h2 class = "articles">
+      Article 11
+    </h2>
+    <ol>
+      <li>
+        Everyone charged with a penal offence has the right to be presumed
+        innocent until proved guilty according to law in a public trial at which he
+        has had all the guarantees necessary for his defence.
+      </li>
+      <li>
+        No one shall be held guilty of any penal offence on account of any act or
+        omission which did not constitute a penal offence, under national or
+        international law, at the time when it was committed. Nor shall a heavier
+        penalty be imposed than the one that was applicable at the time the penal
+        offence was committed.
+      </li>
+    </ol>
+    <h2 class = "articles">
+      Article 12
+    </h2>
+    <p>
+      No one shall be subjected to arbitrary interference with his privacy, family, home
+      or correspondence, nor to attacks upon his honour and reputation. Everyone has
+      the right to the protection of the law against such interference or attacks.
+    </p>
+    <h2 class = "articles">
+      Article 13
+    </h2>
+    <ol>
+      <li>
+        Everyone has the right to freedom of movement and residence within the
+        borders of each State.
+      </li>
+      <li>
+        Everyone has the right to leave any country, including his own, and to
+        return to his country.
+      </li>
+    </ol>
+    <h2 class = "articles">
+      Article 14
+    </h2>
+    <ol>
+      <li>
+        Everyone has the right to seek and to enjoy in other countries asylum from
+        persecution.
+      </li>
+      <li>
+        This right may not be invoked in the case of prosecutions genuinely
+        arising from non-political crimes or from acts contrary to the purposes and
+        principles of the United Nations.
+      </li>
+    </ol>
+    <h2 class = "articles">
+      Article 15
+    </h2>
+    <ol>
+      <li>
+        Everyone has the right to a nationality.
+      </li>
+      <li>
+        No one shall be arbitrarily deprived of his nationality nor denied the right to
+        change his nationality.
+      </li>
+    </ol>
+    <h2 class = "articles">
```

Comment:
пробел добавлять не нужно, исправь везде
```suggestion
    <h2 class="articles">
```